src/main/java/com/incognia/common/AdditionalLocation.java

```diff
@@ -8,5 +9,6 @@
 public class AdditionalLocation {
   Double lat;
   Double lng;
-  Long timestamp;
+  @Deprecated Long timestamp;
+  Instant collectedAt;
```

Comment:
I decided to mark the old field as deprecated and add the new here as this class is used by the user. Given that we are bumping the major due to the removal of `GetSignup`, should we take the opportunity and remove the old field too?

Reply:
sure

Reply:
I'll removed it later together with the other breaking changes.